photon-core/src/main/java/org/photonvision/vision/camera/USBCameraSource.java

```diff
@@ -22,29 +22,35 @@
 import edu.wpi.first.cscore.UsbCamera;
 import edu.wpi.first.cscore.VideoException;
 import edu.wpi.first.cscore.VideoMode;
-import edu.wpi.first.cscore.VideoProperty.Kind;
+import edu.wpi.first.cscore.VideoProperty;
 import edu.wpi.first.util.PixelFormat;
 import java.util.*;
 import java.util.stream.Collectors;
 import org.photonvision.common.configuration.CameraConfiguration;
 import org.photonvision.common.configuration.ConfigManager;
 import org.photonvision.common.logging.LogGroup;
 import org.photonvision.common.logging.Logger;
-import org.photonvision.common.util.TestUtils;
 import org.photonvision.common.util.math.MathUtils;
 import org.photonvision.vision.frame.FrameProvider;
-import org.photonvision.vision.frame.provider.FileFrameProvider;
 import org.photonvision.vision.frame.provider.USBFrameProvider;
 import org.photonvision.vision.processes.VisionSource;
 import org.photonvision.vision.processes.VisionSourceSettables;
 
 public class USBCameraSource extends VisionSource {
     private final Logger logger;
     private final UsbCamera camera;
-    private final USBCameraSettables usbCameraSettables;
-    private FrameProvider usbFrameProvider;
+    protected USBCameraSettables usbCameraSettables;
+    protected FrameProvider usbFrameProvider;
     private final CvSink cvSink;
 
+    private VideoProperty exposureAbsProp = null;
+    private VideoProperty autoExposureProp = null;
+    private double minExposure = 1;
+    private double maxExposure = 80000;
+
+    private int PROP_AUTO_EXPOSURE_ENABLED = 3;
```

Comment:
nit: member is not static final, but is all caps?

Reply:
Yea.... code question time.

It's usually static final. Except for a quirk where we change it. So it can't actually be static final.

I will ruminate on a better way to do this